static/src/javascripts/projects/common/modules/experiments/tests/podcast-container.js

```diff
@@ -0,0 +1,179 @@
+                        seriesUrl: urlWithCampaign(seriesUrl, variant),
+                        audioIcon: addClassesAndTitle(audioIcon.markup, [
+                            `podcast-container-b__audio-icon`,
+                        ]),
+                    });
+
+                    break;
+                case 'c':
+                    oldBody.innerHTML = template(containerCHtml, {
+                        headline,
+                        standfirst,
+                        episodeUrl: urlWithCampaign(episodeUrl, variant),
+                        seriesUrl: urlWithCampaign(seriesUrl, variant),
+                        audioIcon: addClassesAndTitle(audioIcon.markup, [
+                            `podcast-container-c__audio-icon`,
+                        ]),
+                        waveLarge: addClassesAndTitle(
+                            containerWaveCLarge.markup,
+                            [`podcast-container-c__wave-large`]
+                        ),
+                        waveTablet: addClassesAndTitle(
+                            containerWaveCTablet.markup,
+                            [`podcast-container-c__wave-tablet`]
+                        ),
+                        waveMobile: addClassesAndTitle(
+                            containerWaveCMobile.markup,
+                            [`podcast-container-c__wave-mobile`]
+                        ),
+                    });
+
+                    podcastContainer.className +=
+                        ' podcast-container-c__thrasher';
+
+                    break;
+
+                default:
+            }
+        }
+    }
+};
+
+const trackClick = (name: string) => (complete: () => void): void => {
+    const component = document.querySelector(`.${name}`);
+    if (component) {
+        component.onclick = () => {
+            complete();
+        };
+    }
+};
+
+const trackImpression = (name: string) => (track: () => void): void => {
+    const component = document.querySelector(`.${name}`);
+    if (component) {
+        const observer = new window.IntersectionObserver(
+            (entries, self) => {
+                entries.forEach(entry => {
+                    if (entry.isIntersecting) {
+                        self.disconnect();
+                        track();
+                    }
+                });
+            },
+            { threshold: 1.0 }
+        );
+        observer.observe(component);
+    }
+};
+
+export const PodcastContainer = {
+    id: 'PodcastContainer',
+    start: '2018-08-21',
+    expiry: '2018-09-06',
+    author: 'Tom Forbes',
+    description: 'Test designs for a /uk podcasts container',
+    audience: 1,
+    audienceOffset: 0,
+    successMeasure: 'Measure click-through across the variants',
+    audienceCriteria: '',
+    showForSensitive: true,
+    canRun() {
+        return (
+            config.page.pageId === 'uk' && window.CSS.supports('display: grid')
```

Comment:
Ah - i see - cunning: `window.CSS.supports('display: grid')`

Reply:
yep, older browsers will not see the test - though support for grid seems pretty good now